### ISSUE TITLE
publish to `node_modules/.cache/typegraphql` instead of `@generated/typegraphql`

### ISSUE THREAD

**Issue by Morriz** [MichalLytek/typegraphql-prisma#293]

**Comment by MichalLytek** [comment]
@Morriz The problem is that it doesn't work:

![image](https://user-images.githubusercontent.com/10618781/181236243-80189f06-ca0e-46f6-9c4e-77d5bcf877eb.png)

![image](https://user-images.githubusercontent.com/10618781/181236262-917a9825-c025-4340-8641-406757bba503.png)

Cache is for storing artifacts, not JS code to `require` by Node runtime.

**Comment by MichalLytek** [comment]
Or do you think about a facade like `.prisma/client`?

![image](https://user-images.githubusercontent.com/10618781/181236610-5ce1178c-5b38-442b-bdb4-c85f6f3abe54.png)

![image](https://user-images.githubusercontent.com/10618781/181236659-0d26ebf0-835d-4f99-b2ff-57c003139740.png)

![image](https://user-images.githubusercontent.com/10618781/181236697-75a36a95-04e4-4de4-b7f6-aa3e88f4969e.png)

**Comment by Morriz** [comment]
Tnx for looking into it. If a facade already helps to persist the generated files better (I have no clue if it's just vscode that keeps losing them, but like I said it keeps losing ref), then great.

**Comment by MichalLytek** [comment]
You're the first that reports issues with losing that in VSCode.

The current approach of emitting to node_modules is just like installing a npm module.
So that you can just use absolute import of `@generated/xyz`.
Prisma did this at the begining but then switched to fascade some time ago because of issues.

I think I should follow the Prisma approach in that case.
Will try to do this next week or so 😉

**Comment by mwillbanks** [comment]
It's not that you lose them randomly from vscode, anytime you install via npm or yarn it will remove the `@generated` directory and you'll need to run `prisma generate` again.

**Comment by JClackett** [comment]
Yeah having to run the postinstall script every time you add a package is soooo annoying, anyone found a way to avoid this?
